docs/source/tutorial.rst

```diff
@@ -7,47 +7,240 @@ Tutorial
 
-big datasets comment
+Using the method ``validate()`` we can check if the files in the local version are the same than the available canical version,
+and the files were downloaded correctly (none of them are corrupted).
 
+For big datasets: In future ``mirdata`` versions, a random validation will be included. This improvement will reduce validation time for very big datasets.
 
 Accessing annotations
 ^^^^^^^^^^^^^^^^^^^^^
-choice track
-select particular track
-annotation classes and compatibility with jams/mir_eval
 
+We can chose a random track with ``choice_track()`` method.
+
+.. code-block:: python
+
+    random_track = orchset.choice_track()
+    print(random_track)
+    >>> Track(
+           alternating_melody=True,
+           audio_path_mono="user/mir_datasets/orchset/audio/mono/Beethoven-S3-I-ex1.wav",
+           audio_path_stereo="user/mir_datasets/orchset/audio/stereo/Beethoven-S3-I-ex1.wav",
+           composer="Beethoven",
+           contains_brass=False,
+           contains_strings=True,
+           contains_winds=True,
+           excerpt="1",
+           melody_path="user/mir_datasets/orchset/GT/Beethoven-S3-I-ex1.mel",
+           only_brass=False,
+           only_strings=False,
+           only_winds=False,
+           predominant_melodic_instruments=['strings', 'winds'],
+           track_id="Beethoven-S3-I-ex1",
+           work="S3-I",
+           audio_mono: (np.ndarray, float),
+           audio_stereo: (np.ndarray, float),
+           melody: F0Data,
+        )
+
+
+
+We can access to specific tracks by id. The ids are specified in the dataset index.
+In the next example we take the first track of the index, and then we retrieve the melody
+annotation.
+
+.. code-block:: python
+
+    orchset_ids = orchset.track_ids  # Load list of track ids of the dataset
+    orchset_data = orchset.load_tracks()  # Load dataset tracks
+    example_track = orchset_data[orchset_ids[0]]  # Get first track of the index
+
+    # Accessing to track melody annotation
+    example_melody = example_track.melody
+
+
+Alternatively, we don't need to load the whole dataset to get a single track.
+
+.. code-block:: python
+
+    orchset_ids = orchset.track_ids  # Load list of track ids of the dataset
+    example_melody = orchset.track(orchset_ids[0]).melody  # Get melody from first track in the index
+
+
+Annotations can also be accessed through ``load_someAnnotation()`` methods.
+
+.. code-block:: python
+
+    orchset_ids = orchset.track_ids  # Load list of track ids of the dataset
+    example_melody_path = orchset.track(orchset_ids[0]).melody_path  # Parsing melody annotation path
+
+    # Accessing to track melody annotation
+    example_melody = orchset.load_melody(example_melody_path)
+    print(example_melody.frequencies)
+    >>> array([  0.   ,   0.   ,   0.   , ..., 391.995, 391.995, 391.995])
+    print(example_melody.times)
+    >>> array([0.000e+00, 1.000e-02, 2.000e-02, ..., 1.244e+01, 1.245e+01, 1.246e+01])
```

Comment:
I am wondering if this should go in the "working with remote datasets" section exclusively, since we want to favor  usage of the attributes directly? Let's discuss this

Reply:
Yes, IMO I think it's good to inform that the load_ functions can be used if needed, but that the main idea is to retrieve the annotations using the attributes of the dataset. Maybe we could add a minor point somewhere commenting the use of load_ functions and take it away from here.